languagetool-office-extension/src/main/java/org/languagetool/openoffice/LORemoteLanguageTool.java

```diff
@@ -478,6 +505,48 @@ static class RemoteRule extends Rule {
         setOfficeDefaultOff();
       }
       isDictionaryBasedSpellingRule = ruleMap.containsKey("isDictionaryBasedSpellingRule");
+      if (ruleMap.containsKey("ruleOptions")) {
+        Object o = ruleMap.get("ruleOptions");
+        @SuppressWarnings("unchecked")
+        List<Map<String,Object>> rOptions = (List<Map<String, Object>>) o;
+        ruleOptions = new RuleOption[rOptions.size()];
+        for (int i = 0; i < rOptions.size(); i++) {
+          String defaultType = (String) rOptions.get(i).get("defaultType");
+          String configureText = (String) rOptions.get(i).get("configureText");
+          Object defaultValue;
+          Object minConfigurableValue;
+          Object maxConfigurableValue;
+          if (defaultType.equals("Integer")) {
+            defaultValue = (int) rOptions.get(i).get("defaultValue");
+            minConfigurableValue = (int) rOptions.get(i).get("minConfigurableValue");
```

Comment:
`minConfigurableValue` etc. should probably be a constant, as it is used several times